src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
```

Comment:
Consider using a relative path for the background image URL to ensure that the image can be resolved correctly regardless of the environment. For example, `background-image: url(../../images/star.svg);`.